networkmanager/traffic.go

```diff
@@ -299,12 +335,11 @@ func (monitor *trafficMonitoring) deleteTrafficChain(chain, rootChain string) (e
 	return nil
 }
 
-func (monitor *trafficMonitoring) processTrafficMonitor() error {
+func (monitor *trafficMonitoring) processTrafficMonitor() (err error) {
```

Comment:
I guess this function may not return error.

Reply:
I think it's better to leave the ability to return error in case we have problems with iptables

Reply:
There is an issue with this function then. Even if error occurs, it will be overwritten by next loop iteration and next success operation. I guess it should be rewritten in the following way:
```go
if chainErr := ...; chainErr != nil && err == nil {
    err = chainErr
}
```
It should be done for all error cases in this function. In this case, this function will return first occurred error.